modules/claims_api/app/controllers/concerns/claims_api/v2/power_of_attorney_validation.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: false
+
+module ClaimsApi
+  module V2
+    module PowerOfAttorneyValidation
+      def validate_form_2122_and_2122a_submission_values
+        validate_claimant
+        # collect errors and pass back to the controller
+        raise_error_collection if @errors
+      end
+
+      private
+
+      def validate_claimant
+        return if form_attributes['claimant'].blank?
+
+        validate_address
+        validate_relationship
+      end
+
+      def validate_address
+        address = form_attributes.dig('claimant', 'address')
+
+        if address.nil?
+          collect_error_messages(
+            source: '/claimant/address/',
+            detail: "If claimant is present 'adress' must be filled in " \
+                    ' with required fields addressLine1, city, stateCode, country and zipCode'
+          )
+        end
+
+        return if address.nil?
+
+        validate_address_line_one(address)
+        validate_address_city(address)
+        validate_address_state_code(address)
+        validate_address_country(address)
+        validate_address_zip_code(address)
```

Comment:
could `return if address.nil?` be removed and the address validation calls be moved into an `else` clause to avoid checking nil twice?

Reply:
oh ya for sure, good call fixing now